cards/en/swsh12pt5.json

```diff
@@ -8279,7 +8282,10 @@
     "id": "swsh12pt5-153",
     "name": "Fire Energy",
     "supertype": "Energy",
-    "number": "153",
+    "subtypes": [
+      "Basic"
+    ],
+    "number": "152",], "153",
```

Comment:
Looks like some mistakes copy/paste errors perhaps appeared throughout the file.

Reply:
I think I fixed everything, thanks